src/nacl/bindings/crypto_kx.py

```diff
@@ -35,25 +35,41 @@
 crypto_kx_SESSION_KEY_BYTES = lib.crypto_kx_sessionkeybytes()
 
 
-def crypto_kx_keypair(seed=None):
+def crypto_kx_keypair():
     """
     Generate a keypair.
+    This is a duplicate crypto_box_keypair, but
+    is included for api consistency.
+    :return: (public_key, secret_key)
+    :rtype: (bytes, bytes)
+    """
+    public_key = ffi.new("unsigned char[]", crypto_kx_PUBLIC_KEY_BYTES)
+    secret_key = ffi.new("unsigned char[]", crypto_kx_SECRET_KEY_BYTES)
+    res = lib.crypto_kx_keypair(public_key, secret_key)
+    ensure(res == 0, "Key generation failed.", raising=exc.CryptoError)
+
+    return (ffi.buffer(public_key, crypto_kx_PUBLIC_KEY_BYTES)[:],
+            ffi.buffer(secret_key, crypto_kx_SECRET_KEY_BYTES)[:])
+
+
+def crypto_kx_seed_keypair(seed):
+    """
+    Generate a keypair with a given seed.
+    This is a duplicate crypto_box_seed_keypair, but
```

Comment:
This comment can be fixed later; while the functionality is duplicated,  the construction is different, since crypto_kx_seed_keypair() uses the blake2b hash primitive instead of sha512.